monorepo/README.md

```diff
@@ -0,0 +1,80 @@
+# Turborepo starter with Rollup
```

Comment:
@SarjuHansaliya can we include documentation about publishing to npm

Reply:
yes @HariSeldon23 , still working on all those things